### PR TITLE
Add support to import/export OPML file

### PR DESCRIPTION
**Pull request by fasheng**
Add elfeed-org-import-opml and elfeed-org-export-opml and fill the unit tests.

Thanks for such great idea to mange feeds through org for elfeed, it's really conveient for me :)

But it's really a hard work for me when importing and managing my hundreds of feeds from OPML at first. So it will be better that if elfeed-org has it's own import/export methods, wish the code helps~

### CODE REVIEW

**Comment by remyhonig** [comment]
Thank you! Nice idea! I have some small comments I will put in a review. The same applies to this PR as the other in that I'll gladly pick up my own remarks, but it will take some time. If you are willing to make the changes I would really appreciate it. Thanks again for your PR and also thanks again for the tests!

File: elfeed-org.el [hunk 1]
```diff
@@ -36,6 +36,8 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'cl-lib)
+(require 'xml)
```
Comment (remyhonig) [inline elfeed-org.el]: Have you considered to add these packages to the `Package-Requires` header on line 6?  Or are these always available?

Comment (fasheng) [inline elfeed-org.el]: Got it, will update the code soon.

Comment (fasheng) [inline elfeed-org.el]: Searching melpa packages under my .emacs.d just now, looks only `cl-lib` should be append to `Package-Requires`, I'd like to use `(cl-lib "0.5")` like most did, but my is `1.0`. I will check the difference later.

Comment (fasheng) [inline elfeed-org.el]: Updated now, but I found `elfeed` even didn't require `cl-lib` through full of `cl-loop` in it's code, so maybe as a bultin library, requre or not should be all OK~

Comment (fasheng) [inline elfeed-org.el]: I remembered, `Package-Requires` is not necessary at all, melpa will fill it automatically 😂

**Comment by remyhonig** [comment]
merged as #31

**Pull request closed without merging**